tests/unit/Router.ts

```diff
@@ -830,38 +843,6 @@ suite('Router', () => {
 		assert.throws(start, /start can only be called once/);
 	});
 
-	test('start() ensures the previous dispatch is canceled', () => {
```

Comment:
now initial dispatches are sync, setting history after the start call will not need to be cancelled.